packages/manager/apps/order-tracking/index.js

```diff
@@ -0,0 +1,51 @@
+import 'script-loader!jquery'; // eslint-disable-line
+
+import angular from 'angular';
+import ovhManagerCore from '@ovh-ux/manager-core';
+import ssoAuth from '@ovh-ux/ng-ovh-sso-auth';
+import orderTracking from '@ovh-ux/order-tracking';
+
+import 'ovh-ui-kit/dist/oui.css';
+import 'ovh-ui-kit-bs/dist/ovh-ui-kit-bs.css';
+
+angular.module('orderTrackingApp', [
+  orderTracking,
+  ovhManagerCore,
+  'oui',
+  ssoAuth,
+]).config(/* @ngInject */($stateProvider) => {
+  $stateProvider
+    .state('home', {
+      url: '',
+      controller: /* @ngInject */ function ctrl($state) {
+        this.onSubmit = function onSubmit() {
+          $state.go('app', { orderId: this.orderId });
+        };
+      },
+      controllerAs: '$ctrl',
+      template: `<form class="m-4" novalidate data-ng-submit="$ctrl.onSubmit()">
+        <oui-field label="Order ID" size="xl">
+          <input class="oui-input" name="orderId" type="text" data-ng-model="$ctrl.orderId" placeholder="orderId">
+        </oui-field>
+        <oui-button type="submit" disabled="!$ctrl.orderId">Got to tracking</oui-button>
+      </form>`,
+    })
+    .state('app', {
+      url: '/:orderId',
+      template: `
+        <order-tracking-component order-id="$ctrl.orderId"
```

Comment:
I know it could sound overkill given the amount of code but could those be extracted to separate files ?

Reply:
Some remark from me, not meant to be scalable, or maybe I'm missing something about the point of `apps`

Reply:
yes the /apps/order-tracking is just the standalone app showing the order-tracking module ; it's not meant to be reused